tests/test_Galactic.py

```diff
@@ -76,7 +76,19 @@ def test_proper_motion(self):
             "Applying proper motion for +100d failed with separation %.1e arcsec"
             % sep.arcsec
         )
-        assert sep < 1e-6 * u.arcsec, msg
+        assert sep < 1e-9 * u.arcsec, msg
+
+        # sanity check that evaluation at POSEPOCH returns something very close to 0
+        J0613_icrs = self.modelJ0613.coords_as_ICRS()
+        J0613_icrs_alt = self.modelJ0613.coords_as_ICRS(
+            epoch=self.modelJ0613.POSEPOCH.quantity.mjd
+        )
+        sep = J0613_icrs_alt.separation(J0613_icrs)
+        msg = (
```

Comment:
Is this message needed? `pytest` gives pretty extensive context when failures occur, and in particular the test name (no human is calling the function, no need for a conveniently short name).

Reply:
I would have argued yes, but now that I've split it into a separate method, context should be sufficient, so deleted!